apps/leo_gateway/server_key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Don't check in private keys in into the repo, or motivate in the README.md why it is needed.